quick-guide-for-senior-developers.md

```diff
@@ -0,0 +1,80 @@
+
+# The Way We Work: Quick Guide for Senior Developers
+
+## Your Role
+
+Your role is to use software engineering best practices to support rapid and safe delivery of high-quality software.
+
+## Your Focus
+
+At this stage in your career you should focus on the success and professional development of:
+
+- your direct reports (i.e. the people who you line manage)
+- the people within your Delivery Team
+
+## Your Responsibilities
+
+Your key responsibilities are to:
+
+- Lead **System Design** activities within a Delivery Team
+- Ensure there is sufficient **Documentation** to support our products and systems
+- Attend SDMT to assist with **SDMT Operational Planning**
+- Carry out deliberate **Mentoring** to help others develop their knowledge and skills
+- Undertake your **Line Management** duties, in accordance with guidance
+
+When the above responsibilities are met, you may also:
+
+- Carry out **Feature Development** work within a Delivery Team
+- Complete **20% Time** work within each Delivery Team sprint
+- **Pair** with other members of a Delivery Team where this would be productive
+- **Buddy** with newer developers to help them settle into their role
+
+Details on how these responsibilities should be carried out can be found in The Way We Work.
+
+## How You Will Be Supported
+
+Your line manager will:
+
+- meet with you on the first day of each new sprint
+- ensure you have appropriately challenging work to undertake
+- help you record your achievements and plan for upcoming challenges
+- help you choose your own Objectives & Key Results (OKRs)
+- ensure you are using 20% Time to make continuous progress towards your OKRs
+- help you to support the people who you line manage
+
+Within a delivery team:
+
+- Principal Engineers will provide guidance on Engineering Quality Framework standards
+
+As a member of SDMT:
+
+- you will be able to rely on support and advice from other Senior Developers
+- you will receive coaching suppport provided by a Principal Engineer
+
+## How You Will Support Others
+
+As a line manager you will:
+
+- meet with your reports on the first day of each new sprint
+- ensure your reports have appropriately challenging work to undertake
+- help your reports record their achievements and plan for upcoming challenges
+- help your reports choose their own Objectives & Key Results (OKRs)
+- ensure your reports are using 20% Time to make continuous progress towards their OKRs
+
+To support a delivery team you will:
+
+- ensure there is an understandable system design that conforms to approved NHSUK architecture patterns
+- ensure there is sufficient documentation to help others succeed in unfamiliar territory
+- escalate significant issues to SDMT and ensure these issues are resolved
+- deliberately mentor other developers to train them in new areas
+- ensure the team follows Engineering Quality Framework standards and best practices
+
+To support the whole NHSUK team you will:
```

Comment:
To support the whole NHSUK team you should:

or

To support the whole NHSUK team you are required to: